### DISCUSSION TITLE
[Question] Top Axis label on the top data points in line chart.

### DISCUSSION THREAD

**Discussion by MamikonP** [patrykandpatrick/vico#429] [Ideas]
Is it possible to align top axis values to "center". Like I want to align x-axis values on the top of my each data point? Please see the example I want to reach.

**Actual**
![photo_2023-03-02_12-47-02](https://user-images.githubusercontent.com/49882479/222377735-639cadc0-cc6d-45b6-ba00-1d385b0693ed.jpg)

**Expected**
![photo_2023-03-02_12-56-45](https://user-images.githubusercontent.com/49882479/222380141-1874cf6d-90c2-4930-87b7-28a1c2661983.jpg)

**Comment by patrickmichalik** [comment]
Hello! I’d say that wouldn’t exactly be an axis anymore, but rather an extra set of data labels. At the moment, only one data label can be displayed per chart segment, but we’ll look into making it possible to add more data labels, which would enable you to achieve the desired result. Thank you for the suggestion!

**Comment by MamikonP** [comment]
Thank you so much. I'm looking forward to these updates.

**Comment by MamikonP** [comment]
Hi @patrickmichalik I hope you are doing well. Any updates about this enhancement?

**Comment by patrickmichalik** [comment]
Hello! Because this is a FOSS project that we build it in our free time, I’m afraid we don’t provide timelines for changes. Once a feature has been added, the corresponding issue is commented on and closed. We’re open to pull requests.
